css-color-4/Overview.bs

```diff
@@ -1819,7 +1819,7 @@ System Colors</h3>
 	<wpt>
 		parsing/color-valid.html
 		system-color-compute.html
-		system-color-hightlights-vs-getSelection-001.html
```

Comment:
This is indeed the test name. The typo should probably be fixed in WPT, but it is currently correct.